src/client/containers/BoardPage/MemberBoardPage.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import './BoardPage.style.css';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import { useBoard } from './BoardProvider';
+import { useUser } from '../../firebase/UserContext';
+import MemberBoardPageHeader from './BoardPageComponents/BoardHeader/MemberBoardPageHeader';
+import CandidateListPostDeadline from './BoardPageComponents/CandidateList/CandidateListPostDeadline';
+import CandidateListPreDeadline from './BoardPageComponents/CandidateList/CandidateListPreDeadline';
+import ResultButtonPostDeadline from './ResultButton/ResultButtonPostDeadline.container';
+import ResultButtonPreDeadline from './ResultButton/ResultButtonPreDeadline.container';
+import MembersModal from '../../components/ModalViewComponent/MembersModal.component';
+import members from '../../components/ModalViewComponent/membersData.json';
+
+export default function MemberBoardPage() {
+  const { boardInfo } = useBoard();
+  const boardId = boardInfo.id;
+  const { user } = useUser();
+  const userId = user[0].id;
+  const [showModal, setShowModal] = useState(false);
+  const closeModal = () => setShowModal(false);
+  const displayDelete = 'hidden';
+
+  return (
+    <div className="ModalView">
+      {showModal && (
+        <div
+          onClick={closeModal}
+          onKeyDown={closeModal}
+          className="back-drop"
+          aria-hidden="true"
+        />
+      )}
+      <div className="Board-container">
+        <div className="Header-component">
+          <MemberBoardPageHeader />
+        </div>
+        <div className="title">
+          <PageTitle title={boardInfo.title} />
+          <GenericButton
+            className="generic-button"
+            buttonSize="small"
+            buttonType="secondary"
+            buttonDisabled={false}
+            onClick={() => setShowModal(true)}
+            buttonLabel="Members"
+          />
+        </div>
+        <MembersModal show={showModal} close={closeModal} members={members} />
+        {boardInfo.hasPassedDeadline() ? (
+          <div>
+            <div className="CandidateCard-component-member">
```

Comment:
Is there a reson to put all these in divs? The components should contain the necessary styling